cmd/create/cluster/cmd.go

```diff
@@ -3348,6 +3348,11 @@ func minReplicaValidator(multiAZ bool, isHostedCP bool, privateSubnetsCount int)
 			return err
 		}
 
+		if isHostedCP && minReplicas < 2 {
+			return fmt.Errorf("hosted Control Plane clusters require a minimum of 2 nodes, "+
```

Comment:
The minimum of nodes varies from single to multi AZ, and the min replicas variable usually holds the value for that. Should this be validating what's supplied in args.computeNodes instead?

Reply:
While it applies for Hive, the MinReplicaValidator doesn't check multiAZ minimal number of replicas for hosted CP https://github.com/openshift-online/ocm-common/blob/main/pkg/cluster/validations/cluster_node.go#L22.
We could validate args.computeNodes, but then it will be possible to supply 0 replicas in the interactive mode resulting in the same bug.

Reply:
I see, thanks!